src/libs/actions/Task.js

```diff
@@ -292,8 +292,11 @@ function reopenTask(taskReportID, taskTitle) {
  * @param {object} report
  * @param {string} ownerEmail
  * @param {Number} ownerAccountID
- * @param {{title?: string, description?: string, assignee?:string, assigneeAccountID?:number}} editedTask
- * @returns {object} action
+ * @param {Object} editedTask
+ * @param {String} editedTask.title
+ * @param {String} editedTask.description
+ * @param {String} editedTask.assignee
+ * @param {Number} editedTask.assigneeAccountID
```

Comment:
Ooh i like this better, did lint recommend this or did you just fix it?

Reply:
Lint was throwing out error on this, I fixed it (it would be cool if it did automatically 👀 )

Reply:
Yeah right, i bet `prettier` could automatically write docs 🤔  maybe wishful thinking lol